src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
> The default shout value is about 1.5 to 2 in-game houses in distance. As a young kid, I'd shout to a friend 5 houses down.

Don't compare too much to real life ranges. 6 is maximum shotgun range, 14 maximum smg range. It takes almost 2.5 minutes to walk the length of a single house.

> A shout should draw many nearby hordes.

> NPC chatting distance is 24 tiles. Having shout limited to a comparable distance of ~30 makes this feature near useless.

A shout that would draw more hordes than NPCs sounds even less useful.

NPC talk range could be shortened, since it is also the trade range and it was set to 24 long ago, when you had to catch up to NPCs to talk to them. Also, you could use it to draw NPCs around houses and in the night.

But yeah, a shout command to draw neutral NPCs near will have to be nearly useless, unless it somehow allowed drawing them from overmap or having them stop combat to respond to the shout.